packages/babel-runtime/package.json

```diff
@@ -6,12 +6,12 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-runtime",
   "author": "Sebastian McKenzie <sebmck@gmail.com>",
   "dependencies": {
-    "core-js": "^2.1.0"
+    "core-js": "^2.1.0",
+    "regenerator-runtime": "^0.9.5"
```

Comment:
Should this be an exact version constraint? I don't want minor `regenerator-runtime` changes to risk breaking `babel-runtime`.